#array with all the modem ids:
mod_id: [1,2]

